src/renderer/components/ft-prompt/ft-prompt.js

```diff
@@ -64,6 +67,9 @@ export default defineComponent({
     })
     this.focusItem(0)
   },
+  destroyed() {
+    this.lastActiveElement?.focus()
+  },
```

Comment:
`destroyed` is called after the component instance has been destroyed, so `this` is no longer available at that point. `beforeDestroy` is called just before the component instance is destroyed, so `this` is still available.
```suggestion
  beforeDestroy() {
    this.lastActiveElement?.focus()
  },
```